packages/@okta/vuepress-site/use_cases/inline_hooks/saml_hook/saml_hook/index.md

```diff
@@ -0,0 +1,391 @@
+
+### commands
+
+The `commands` object is where you can tell Okta to add additional claims to the assertion or to modify the existing assertion statements.
+
+`commands` is an array, allowing you to send multiple commands. In each array element, there needs to be a `type` property and `value` property. The `type` property is where you specify which of the supported commands you wish to execute, and `value` is where you supply an operand for that command.
+
+In the case of the SAML Assertion Inline Hook, the `value` property is itself a nested object, in which you specify a particular operation, a path to act on, and a value.
+
+| Property | Description                                                              | Data Type       |
+|----------|--------------------------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands).                    | String          |
+| value    | Operand to pass to the command. It specifies a particular op to perform. | [value](#value) |
+
+#### Supported Commands
+
+The following command is currently supported for the SAML Assertion Inline Hook type:
+
+| Command                  | Description              |
+|--------------------------|--------------------------|
+| com.okta.assertion.patch | Modify a SAML assertion. |
+
+#### value
+
+The `value` object is where you specify the specific operation to perform. It is an array, allowing you to request more than one operation.
+
+| Property | Description                                                                                                                                                                                                           | Data Type       |
+|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| op       | The name of one of the [supported ops](#list-of-supported-ops).                                                                                                                                                       | String          |
+| path     | Location, within the assertion, to apply the operation. See [Specifying Location within the Assertion](##specifying-location-within-the-assertion) below. | String          |
+| value    | Value to set the claim to.                                                                                                                                                                                            | Any JSON object |
+
+#### List of Supported Ops
+
+| Op      | Description                             |
+|---------|-----------------------------------------|
+| add     | Add a new claim to the assertion.       |
+| replace | Modify an existing attribute statement. |
```

Comment:
`replace` can modify _anything_ in the assertion object, not just attribute statements